packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -147,41 +147,6 @@ describe("SharedTree", () => {
             }
         });
     });
-
-    it("can edit using editable-tree", async () => {
```

Comment:
I think it was accidental, as my VS Code didn't list this change before I pushed. I'll put it back.

Reply:
@CraigMacomber Restored, but without `anchorSymbol`, which seems to be not needed anymore as an editing is implemented here, to some extent at least.
However, as more I think about that, more I get to the point, that it might actually be very useful to be able to get a path for a node of EditableTree. What do you think?